src/shogun/classifier/svm/LibSVM.cpp

```diff
@@ -15,29 +15,42 @@
 using namespace shogun;
 
 CLibSVM::CLibSVM()
-: CSVM(), model(NULL), solver_type(LIBSVM_C_SVC)
+: CSVM(), solver_type(LIBSVM_C_SVC)
```

Comment:
you cannot use the initializer in here if you call register/init parameters after (works as it is now, but would be better to init everything in the registration method)